dev/io.openliberty.persistence.3.2.thirdparty/resources/org/eclipse/persistence/internal/localization/i18n/ExceptionLocalizationResource.nlsprops

```diff
@@ -149,6 +163,8 @@ file_exists=The file {0} already exists.
 
 file_not_exist=File {0} does not exist.
 
+find_option_class_unknown=The FindOption implementing class {0} is not supported
```

Comment:
Change "class {0}" to "the {0} class" unless the class name value will be so long to make it difficult to read